packages/extension-ui/src/partials/MenuSettings.tsx

```diff
@@ -33,6 +33,7 @@ function MenuSettings ({ className, reference }: Props): React.ReactElement<Prop
   const [prefix, setPrefix] = useState(`${settings.prefix === -1 ? 42 : settings.prefix}`);
   const themeContext = useContext<Theme>(ThemeContext);
   const setTheme = useContext(ThemeSwitchContext);
+  const { isPopup } = useIsPopup();
```

Comment:
```suggestion
  const isPopup = useIsPopup();
```

Adjusted with hook suggestion above.